src/pages/contracts/_components/VerifiedContractRow.tsx

```diff
@@ -1,135 +1,191 @@
+import { SmartContractProps } from "@api/types";
 import LinkText from "@components/commons/LinkText";
-import { VerifiedContract } from "mockdata/VerifiedContractData";
+import clsx from "clsx";
+import { formatDateToUTC } from "shared/durationHelper";
 import { truncateTextFromMiddle } from "shared/textHelper";
+import { FiFileText } from "react-icons/fi";
+import BigNumber from "bignumber.js";
+
+interface ListTitleProps {
+  title: string;
+  className?: string;
+}
+
+function ListTitle({ title, className }: ListTitleProps) {
+  return (
+    <div className={clsx("tracking-[0.01em] text-white-700", className)}>
+      {title}
+    </div>
+  );
+}
+
+interface ListBodyProps {
+  body: string;
+  className?: string;
+}
+
+function ListBody({ body, className }: ListBodyProps) {
+  return (
+    <div
+      className={clsx("tracking-[0.01em] text-white-50 break-all", className)}
+    >
+      {body}
+    </div>
+  );
+}
+
+const fixedTitle = {
+  contractName: "Contract name",
+  version: "Version",
+  compiler: "Compiler",
+  transactions: "Transactions",
+  verified: "Verified",
+};
 
 export default function VerifiedContractRow({
   data,
 }: {
-  data: VerifiedContract;
+  data: SmartContractProps;
 }) {
   return (
     <>
       {/* desktop */}
       <div
-        data-testid={`${data.contract}-desktop-verified-list`}
+        data-testid={`${data.address.hash}-desktop-verified-list`}
         className="hidden lg:block"
```

Comment:
better to just use desktop view for extra large screens to prevent word breaks on `lg` screens 
```suggestion
        className="hidden xl:block"
```

Reply:
if we are using `lg:block` then it will be visible for lg and xl screen